Sources/Core/Other/TextViewMonitor.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  TextViewMonitor.swift
```

Comment:
As specified in Issue #23, this class should be named `TextViewTextMonitor` (because it is monitoring the _text_ associated with a text view).